lms/djangoapps/instructor/enrollment_report.py

```diff
@@ -0,0 +1,56 @@
+"""
+Defines abstract class for the Enrollment Reports.
+"""
+from instructor.enrollment_report_provider import EnrollmentReportProvider
+from django.contrib.auth.models import User
+import collections
+import json
+
+
+class BaseEnrollmentReportProvider(EnrollmentReportProvider):
```

Comment:
It seems that this class should be merged with `EnrollmentReportProvider`. Aside from tests there doesn't seem to be a case where we would not always use `BaseEnrollmentReportProvider`.